tests/factories/file.js

```diff
@@ -3,25 +3,33 @@ import faker from 'faker';
 
 FactoryGuy.define('file', {
     default: {
-        name: () => faker.lorem.word() + '.txt',
+        name: () => faker.lorem.words(1) + '.txt',
```

Comment:
Yeah, that's odd- according to `bower ls` and our bower.json, the default is to install 3.0.0-3.0.1. 

But according to `bower_components/Faker/package.js`, I have 2.1.1 locally. Usually bower should trigger a conflict in that case... I've found their dependency resolution mechanisms painful to contemplate in the past.

`words(1)` isn't a hassle; it's just that the newer fakers have some syntactic sugar for generating filenames. Ah well.